recipe-server/normandy/settings.py

```diff
@@ -389,6 +399,13 @@ class Production(Base):
     DEFAULT_FILE_STORAGE = values.Value('normandy.base.storage.S3Boto3PermissiveStorage')
     AWS_S3_FILE_OVERWRITE = False
 
+    # Custom CORS settings that overrides the CORS class's configuration.
+    # In production we harden it down a bit extra.
+    CORS_ORIGIN_ALLOW_ALL = values.BooleanValue(False)
+    CORS_ORIGIN_WHITELIST = values.ListValue([
+        'https://normandy-admin.prod.mozaws.net'
```

Comment:
I think we should leave this list empty by default. We're going to be customizing this per-environment with environment variables anyways, and this value won't actually be needed. Same-origin requests don't use CORS and so don't need any of these headers.

I'd keep the line setting `ALLOW_ALL` to `False` though. That's good.